### PR TITLE
Remove EC2 Dockstore deploy

### PR DESCRIPTION
**Pull request by kathy-t**
**Description**
This PR removes code that supports the EC2 Dockstore deploy. This includes removing the docker-compose file, webservice-digest script, and the Dockerfile used to build the nginx image. There are other things untouched in this repo that aren't used for Fargate Dockstore...not sure if that should be cleaned up. Eventually, it would be nice to collapse this repo into dockstore-deploy.

Also added removing `IS_FARGATE_DEPLOY` from Secrets Manager in the [release checklist](https://github.com/dockstore/dockstore-deploy/wiki/1.14-Staging#pre-deploy).

I haven't updated the README yet, will wait for some initial feedback first.

This PR is a companion to https://github.com/dockstore/dockstore-deploy/pull/643.

**Review Instructions**
Verify that the Fargate Dockstore deploy is successful. Can look in dockstore-dev-info for the nightly deploy message to confirm the deploy in QA succeeded.

**Issue**
[SEAB-4723](https://ucsc-cgl.atlassian.net/browse/SEAB-4723)

Please make sure that you've checked the following before submitting your pull request. Thanks!

- [x] Ensure that the PR targets the correct branch. Check the milestone or fix version of the ticket.

### CODE REVIEW

**Review by coverbeck — APPROVED** [review]

**Comment by svonworl** [comment]
Should we update the `README`?

**Review by svonworl — APPROVED** [review]

**Comment by denis-yuen** [comment]
> There are other things untouched in this repo that aren't used for Fargate Dockstore...not sure if that should be cleaned up.

I think so, with https://github.com/dockstore/dockstore/blob/develop/docker-compose.yml in play, I'm not sure what else should be in here but what we use for Fargate (except for maybe the Kibana stuff which we may resurrect at some point)

File: dockstore_launcher_config/compose.config [hunk 1]
```diff
@@ -39,16 +39,9 @@
 "GOOGLE_CLIENT_ID":"potato",
 "GOOGLE_CLIENT_SECRET":"potato",
 "HTTPS":false,
-"IS_FARGATE_DEPLOY":false,
```
Comment (denis-yuen) [inline dockstore_launcher_config/compose.config:42]: Note sure on these, but the `HTTPS`, `PUBLIC_LAUNCHER_IP_ADDRESS` may  be useless too. Do a search and see what uses them.

**Comment by david4096** [comment]
Seconded on merging compose_setup into -deploy

**Review by david4096 — APPROVED** [review]

File: dockstore_launcher_config/compose.config (hunk 1 above)
Comment (kathy-t) [reply dockstore_launcher_config/compose.config:42]: Removed them

**Pull request merged**